storage/innobase/xtrabackup/doc/source/security/pxb-apparmor.rst

```diff
@@ -0,0 +1,30 @@
+.. _pxb-apparmor:
+
+=============================================
+Working with AppArmor 
+=============================================
+
+The Linux Security Module implements mandatory access controls (MAC) with AppArmor. Debian and Ubuntu systems install AppArmor by default. Apparmor uses profiles which define which files and permissions are needed for application.
+
+|Percona XtraBackup| does not have a profile and is not confined by AppArmor. 
+
+For a list of common AppArmor commands, see `Percona Server for MySQL - AppArmor <https://www.percona.com/doc/percona-server/LATEST/security/apparmor.html>`_
+
+Develop a profile
+-------------------
+
+The easiest 
+
+Download the profile from:
+
+The sections in bold should be completed with your system's information, such as where the backup destination directory is located.
```

Comment:
Which sections are we referring to?
I think this was copied from my original file where I had the full AA profile with bolded parts.

Reply:
Fixed.